src/d3d12shader.rs

```diff
@@ -0,0 +1,325 @@
+// Copyright © 2016; Dmitry Roschin
+// Licensed under the Apache License, Version 2.0, <LICENSE-APACHE or
+// http://apache.org/licenses/LICENSE-2.0> or the MIT license <LICENSE-MIT or
+// http://opensource.org/licenses/MIT>, at your option. This file may not be
+// copied, modified, or distributed except according to those terms.
```

Comment:
Winapi is currently only licensed under MIT. Could you perhaps update this to match what the rest of the source files do?